starboard/android/shared/platform_configuration/BUILD.gn

```diff
@@ -164,12 +164,16 @@ config("speed") {
 }
 
 config("executable_config") {
-  cflags = [ "-fPIE" ]
-  ldflags = [ "-pie" ]
+  if (current_toolchain == default_toolchain) {
```

Comment:
What does it mean when current_toolchian == default_toolchain? Does it mean target/device build?

Reply:
Yes, building for the target and not building a host tool